### PR TITLE
Remove `OpCode.CALL`

### PR DESCRIPTION
**Pull request by erikzhang**
Closes #145

### CODE REVIEW

File: src/neo-vm/ExecutionEngine.cs [hunk 1]
```diff
@@ -277,14 +277,18 @@ private bool ExecuteInstruction()
                                 context.InstructionPointer += 3;
                             return true;
                         }
-                    case OpCode.CALL:
+                    case OpCode.CALL_I:
```
Comment (shargon) [inline src/neo-vm/ExecutionEngine.cs]: Maybe we can rename `CALL_I` to `CALL` ?

Comment (shargon) [reply src/neo-vm/ExecutionEngine.cs]: For me, the logic change by default to be isolated is good, but i think that we can call it as `CALL`

Comment (erikzhang) [reply src/neo-vm/ExecutionEngine.cs]: I am wondering if we really need stack isolation in the case of executing internal calls.

Comment (shargon) [reply src/neo-vm/ExecutionEngine.cs]: Agree with you, but then we should remove `CALL_I` not `CALL`

Comment (erikzhang) [reply src/neo-vm/ExecutionEngine.cs]: Yes. I will confirm with @lightszero.

**Pull request closed without merging**